### PR TITLE
Adding BEIS heat and steam & water supply

### PR DESCRIPTION
**Pull request by patstirling**
**Is this a new addition or an update to an existing factor?**

- [X] New
- [ ] Update

**What category or categories does it affect?**

Energy, Water

**What?**

Adding BEIS values for these categories

**Why?**

_Please give a brief explanation of why these changes are valuable or necessary_

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1520,4 +1525,6 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_closed_loop
...
-Water,Treatment,water-treatment,Water Treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of Water Treatment in the UK in 2021. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,unknown
+Water,Water Supply,water-supply,Water supply,kg-CO2e/m3,0.149,,BEIS,2021,GB,2021/11/25,Emission intensity for water supply. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2023,upstream
+Water,Water Supply,water-supply,Water supply,kg-CO2e/ML,149,,BEIS,2021,GB,2021/11/25,Emission intensity for water supply. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2024,upstream
```
Comment (grindcode) [inline OpenEmissionFactorsDB.csv:1529]: This EF is a duplicated of the previous one, simply in a different unit but same type (Volume). I would argue there is no value in adding this.

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1520,4 +1525,6 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_closed_loop
...
 Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_open_loop,PVC waste disposal (incl. forming) - open-loop,kg-CO2e/t,21.294,,BEIS,2021,GB,2021/11/18,Emission intensity for PVC waste disposal (incl. forming) - open-loop. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/998218/conversion-factors-2021-flat-file-automatic-processing.xls,end_of_life
-Water,Treatment,water-treatment,Water Treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of Water Treatment in the UK in 2021. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,unknown
+Water,Water Supply,water-supply,Water supply,kg-CO2e/m3,0.149,,BEIS,2021,GB,2021/11/25,Emission intensity for water supply. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2023,upstream
```
Comment (grindcode) [inline OpenEmissionFactorsDB.csv:1528]: Source link is broken

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (grindcode) [reply OpenEmissionFactorsDB.csv:1529]: Source link is broken

**Review by grindcode — CHANGES_REQUESTED** [review]
Duplicated EF + broken source links.

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:1529]: Agree. Also, does the source mention any details about the upstream activities?

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:1529]: Have removed duplicate and updated descriptions/lca in line with https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/990675/2021-ghg-conversion-factors-methodology.pdf - please do review!

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:1528]: Links fixed

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -371,65 +371,70 @@ Energy,Fuel,fuel_type_biodiesel_me_used_cooking_oil-fuel_use_na,Biodiesel ME (Fr
...
+Energy,Heat and Steam,heat-and-steam-type_district_wtt,District heat and steam (WTT),kg-CO2e/kWh,0.03153,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - district heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2026,upstream
+Energy,Heat and Steam,heat-and-steam-type_onsite,Onsite heat and steam,kg-CO2e/kWh,0.17073,,BEIS,2021,GB,2021/11/25,Emission intensity for onsite heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,use_phase
+Energy,Heat and Steam,heat-and-steam-type_onsite_wtt,Onsite heat and steam (WTT),kg-CO2e/kWh,0.03153,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - onsite heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2025,upstream
```
Comment (grindcode) [inline OpenEmissionFactorsDB.csv:437]: source link is broken

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -371,65 +371,70 @@ Energy,Fuel,fuel_type_biodiesel_me_used_cooking_oil-fuel_use_na,Biodiesel ME (Fr
...
+Energy,Heat and Steam,heat-and-steam-type_district_wtt_distribution_5percent_loss,District heat & steam (WTT; distribution 5% loss),kg-CO2e/kWh,0.00166,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - district heat & steam distribution 5% loss. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2027,upstream
+Energy,Heat and Steam,heat-and-steam-type_district,District heat and steam,kg-CO2e/kWh,0.17073,,BEIS,2021,GB,2021/11/25,Emission intensity for district heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,use_phase
+Energy,Heat and Steam,heat-and-steam-type_district_wtt,District heat and steam (WTT),kg-CO2e/kWh,0.03153,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - district heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2026,upstream
```
Comment (grindcode) [inline OpenEmissionFactorsDB.csv:435]: source link is broken

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -371,65 +371,70 @@ Energy,Fuel,fuel_type_biodiesel_me_used_cooking_oil-fuel_use_na,Biodiesel ME (Fr
...
 Energy,Fuel,fuel_type_waste_oils-fuel_use_na,Waste oils,kg-CO2e/t,3224.56,,BEIS,2021,GB,2021/11/17,Emission intensity for waste oils. The fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
+Energy,Heat and Steam,heat-and-steam-type_district_wtt_distribution_5percent_loss,District heat & steam (WTT; distribution 5% loss),kg-CO2e/kWh,0.00166,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - district heat & steam distribution 5% loss. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2027,upstream
```
Comment (grindcode) [inline OpenEmissionFactorsDB.csv:433]: source link is broken

**Review by grindcode — CHANGES_REQUESTED** [review]

**Review by grindcode — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -1520,4 +1525,5 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_closed_loop
...
-Water,Treatment,water-treatment,Water Treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of Water Treatment in the UK in 2021. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,unknown
+Water,Water Supply,water-supply,Water supply,kg-CO2e/m3,0.149,,BEIS,2021,GB,2021/11/25,Emission intensity for water supply. Based on emissions associated with offices/transport/treatment of water supply. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
+Water,Water Treatment,water-treatment,Water treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of water treatment in the UK in 2021. Based on emissions associated with offices/transport/treatment of wastewater. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1529]: Looks great! My only comment is that the water treatment here sounds to be more related to the end-of-life processes.

**Review by maryarbab — CHANGES_REQUESTED** [review]

**Review by maryarbab — APPROVED** [review]

**Pull request merged**